symplyphysics/definitions/damped_harmonic_oscillator_equation.py

```diff
@@ -0,0 +1,139 @@
+from sympy import (
+    Derivative,
+    symbols,
+    Function as SymFunction,
+    dsolve,
+    sqrt,
+    exp,
+    cos,
+)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+    dimensionless,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+
+# Description
+## Assuming there is a damping force acting on an oscillating body that is linearly proportional
+## to the body's velocity, we can write a differential equation for the body's position. We're
+## assuming the body only moves in one direction.
+
+# Definition: d**2(x(t))/dt**2 + 2*zeta*omega*d(x(t))/dt + (omega**2)*x(t) = 0
+## x(t) - position of the oscillating body
+## t - time
+## omega - undamped angular frequency
+## zeta - damping ratio, the value of which critically determines the behavior of the system
+
+# not specifying the dimensions because it can be any physical quantity
+displacement = symbols("displacement", cls=SymFunction, real=True)
+time = Symbol("time", units.time, positive=True)
+undamped_angular_frequency = Symbol(
+    "undamped_angular_frequency", angle_type / units.time, positive=True
+)
+damping_ratio = Symbol("damping_ratio", dimensionless, positive=True)
+
+definition = (
+    Derivative(displacement(time), time, time)
+    + 2 * damping_ratio * undamped_angular_frequency * Derivative(displacement(time), time)
+    + undamped_angular_frequency**2 * displacement(time)
+)
+
+# The general solution looks like this:
+## x(t) = exp(-w*z*t) * (C1 * exp(w*sqrt(z**2 - 1)*t) + C2 * exp(-w*sqrt(z**2 - 1)*t))
+
+# z > 1, overdamped system
+## the system exponentially decays to steady state without oscillations
+## larger values of the damping ratio return to equilibrium more slowly
+
+overdamped_displacement = dsolve(definition, displacement(time)).rhs
```

Comment:
This is general solution, not for overdamped case, if I'm getting it right.

Reply:
But the overdamped case looks exactly the same as the general solution. $`z > 1`$ doesn't bring that many interesting properties to the equation that make it stand out somehow

Reply:
Hm, you are right. Probably we should simply put it into the description of general solution and do not extract to separate law.

Reply:
I actually think it might be better to create a directory `laws/kinematics/damped_oscillations` and move the related laws there, and put the general solution there too

Reply:
I suggest `oscillators` folder.